tracex_project/db_results/templates/metrics_pj_overview.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Metrics Overview</title>
+        {% load static %}
+        <link rel="stylesheet" type="text/css" href="{% static '/tracex/css/style.css' %}">
+    </head>
+<body class="main_body">
+    <img src="{% static '/tracex/img/result_dashboard_logo.png' %}" alt="Metrics Dashboard" class="metrics_dashboard_logo">
```

Comment:
same problem with the icon as in the db_results_overview.html
![grafik](https://github.com/bptlab/TracEX/assets/116206992/5e8262d7-7d5d-4625-b08a-17e8dd951ce7)

Reply:
your browser is using the style.css from ur cache thats why the design is off. I will resolve this thread for now, please reopen if u need assistance